src/main/java/uk/gov/hmcts/reform/judicialapi/controller/JudicialController.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.judicialapi.controller;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.reform.judicialapi.controller.response.JudicialRoleTypeEntityResponse;
+import uk.gov.hmcts.reform.judicialapi.service.JudicialRoleTypeService;
+
+@RequestMapping(path = "refdata/v1/judicial")
+@RestController
+@Slf4j
+@NoArgsConstructor

```

Comment:
do we really required @NoArgConstructor ?

Reply:
I don't believe we do. I have removed it.